drud/ddev-contrib
# Add PHP 8.1 prerelease support

## The New Solution/Problem/Issue/Bug:

PHP 8.1 prerelease support

## How this PR Solves The Problem:

Uses an external php container with 8.1.0alpha1 in it.

## Manual Testing Instructions:

Try out the instructions in the README. Use a phpinfo.php to see what's happening.

## Related Issue Link(s):
